Utilities/bash/completions

```diff
@@ -114,11 +114,11 @@ _swift_package()
 _swift_test()
 {
     case $prev in
-        (-s|--specifier)
```

Comment:
lets leave these file be, they're highly outdated. We should just remove them in a separate PR.